### PR TITLE
Weak topology is Hemicompact more than σ-compact

### PR DESCRIPTION
**Pull request by yhx-12243**

### CODE REVIEW

**Comment by prabau** [comment]
There a few grammar issues. Also, sorry but there is also sloppy language in what you added.  One can guess what you are trying to say.  But it would not be clear to someone who doesn't already know the result.  e.g., what is a subset of what, what topology, is it the norm topology, is the weak topology, etc, etc.  What does "pointwise bounded" even mean a priori for a subset of a Hilbert space, etc, etc.  Can you try to make this clearer?

**Comment by yhx-12243** [comment]
Because it is a proposition basically on functional analysis rather than topology. So the language will tend to in the flavor of functional analysis (such like pointwise bounded).

**Comment by prabau** [comment]
Thanks, this is better.

It is clear what it means for a set of linear operators, or of linear functionals for example, to be pointwise bounded.  What I was objecting to was the use of "pointwise bounded" for a set of vectors.  That makes no sense in general, unless like here one identifies each $x\in H$ with the corresponding functional "inner product with $x$".
That's one of the reasons I thought the previous version was a little sloppy.

What terminology do people use normally in functional analysis to say that a set of vectors in $H$ is "pointwise bounded" in this sense?

**Comment by yhx-12243** [comment]
Yes, it refers to norm topology. I'm sorry about using "pointwise bounded" because I automatically embedding it to the double dual in my brain (since it is a relatively common technique)

**Comment by david20000813** [comment]
Apologies for just having time now to review this. I was busy the past few days grading finals and arranging for my holiday travels.

> Would "pointwise bounded" also be called "weakly bounded" (or maybe it should be "weak-* bounded", not sure) in this case?
> 
> In any case, maybe @david20000813 can review, as he has a lot of experience with functional analysis.
> 
> But thanks for the result. It's a good improvement.

I think it is better to call it "bounded in weak topology". There is a specific notion of "boundedness" in a topological vector space, which is exactly what pointwise boundedness indicates here. I'd say, to me, pointwise bounded would be understandable as a functional analyst, but it definitely is not standard terminology.

File: spaces/S000021/properties/P000111.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  name: Functional Analysis
+---
+By the Banach-Alaoglu theorem (see Theorem 5.2.1 of
+{{doi:10.1007/978-93-86279-42-2}}), the closed unit ball (with respect
+to the inner product of $H$) is compact in the weak-* topology, which
+on a Hilbert space is equivalent to the weak topology.  Since $H$ is a
+topological vector space, the same is true for every closed ball.
+
+Let $K_n$ be the closed ball centered at origin with radius $n$, then $K_n$ is compact and $H = \bigcup_{n \in \omega} K_n$.
+Suppose $K$ is any (weakly) compact subset of $H$. From the fact that the continuous image of a compact set is compact, we know that $K$ is pointwise bounded, namely $\sup_{y \in K} \left| \left< x, y \right> \right| < + \infty$ for any $x \in H$.
```
Comment (david20000813) [inline spaces/S000021/properties/P000111.md:16]:
```suggestion
Suppose $K$ is any (weakly) compact subset of $H$. From the fact that the continuous image of a compact set is compact, we know that $K$ is bounded in the weak topology, that is, $\sup_{y \in K} \left| \left< x, y \right> \right| < + \infty$ for any $x \in H$.
```

File: spaces/S000021/properties/P000111.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Therefore, $K$ is bounded (in strong topology) by [Uniform Boundedness Principle](https://en.wikipedia.org/wiki/Uniform_boundedness_principle), thus contained in some $K_n$.
```
Comment (david20000813) [inline spaces/S000021/properties/P000111.md:18]:
```suggestion
Therefore, $K$ is bounded in norm by [Uniform Boundedness Principle](https://en.wikipedia.org/wiki/Uniform_boundedness_principle), thus contained in some $K_n$.
```

**Review by david20000813 — APPROVED** [review]

**Pull request merged**